src/usr/lib.ar71xx/ar71xx.json-cut

```diff
@@ -35,9 +35,9 @@
 			"gpios": [ 0, 1 ],
 			"codes": [ "BTN_0", "BTN_1" ],
 			"positions": {
-				"lo_hi": "left",
-				"lo_lo": "center",
-				"hi_lo": "right"
+				"lo_hi": "switch-left",
+				"lo_lo": "switch-center",
+				"hi_lo": "switch-right"
```

Comment:
This isn't necessary. The final button name is a combination of the switch name (`switch` in this case, from line 34) and the position names.

If this was incorrect, then the button names in your logs would have also been incorrect.